abis/DAOFactory.json

```diff
@@ -0,0 +1,367 @@
+[
```

Comment:
I can see it is convenient to have the ABIs in this repository, but they are copied from somewhere else, and copying code is usually a sign something is wrong :-)

In this case, what is wrong is that it is not explicit with which arc versions this CommonFactory version is expected to work

Reply:
Because right now don't have any final version to use for common, so I took the latest and will update as there are new changes.

Reply:
@ben-kaufman can you import arc or migration(which also include the Abis ) to this repo and exclude it form the npm package somehow ?

Reply:
The migration is already there for the test, but I don't think this is what @jellegerbrandy meant.